packages/manager/modules/user-account/src/user/advanced/advanced.routing.js

```diff
@@ -2,6 +2,6 @@ export default /* @ngInject */ ($stateProvider) => {
   $stateProvider.state('app.account.user.advanced', {
     url: '/advanced',
     component: 'accountUserAdvanced',
-    translations: ['../'],
+    translations: ['../', './'],
```

Comment:
We should maybe prefer an annotation
```suggestion
    .run(/* @ngTranslationsInject:json ./translations */)
```